support-frontend/app/services/stepfunctions/StateWrapper.scala

```diff
@@ -31,11 +32,11 @@ class StateWrapper(encryption: EncryptionProvider, useEncryption: Boolean) {
       decoded <- decodeState(unwrapped.state)(decoder)
     } yield decoded
 
-  private def encodeState[T](state: T)(implicit encoder: Encoder[T]): String = encodeToBase64String(encryption.encrypt(state.asJson.noSpaces))
+  private def encodeState[T](state: T)(implicit encoder: Encoder[T]): String = encodeToBase64String(state.asJson.noSpaces.getBytes(utf8))
 
   private def decodeState[T](state: String)(implicit decoder: Decoder[T]): Try[T] = for {
     state <- Try(Base64.getDecoder.decode(state))
-    decrypted <- Try(encryption.decrypt(state))
+    decrypted <- Try(new String(state, utf8))
```

Comment:
You can use `java.nio.charset.StandardCharsets.UTF_8` so you don't have to have literal values.